docs/getting-started.md

```diff
@@ -48,7 +48,6 @@ We use [this script](https://github.com/allenai/dolma/blob/main/scripts/make_wik
 ```shell
 python scripts/make_wikipedia.py \
   --output wikipedia \
-  --languages simple \
```

Comment:
Why changing this line?

Reply:
hi, we had already provided `--lang simple`